### PR TITLE
CB-8936 Gathering logs on Windows

### PR DESCRIPTION
**Pull request by alsorokin**
https://issues.apache.org/jira/browse/CB-8936

### CODE REVIEW

**Comment by nikhilkh** [comment]
This seems useful. If it works well you should add it to cordova-windows with a 'log' script, similar to how other platforms do it and just invoke it from there.

File: medic/medic-run.js [hunk 1]
```diff
@@ -232,12 +233,18 @@ function windowsSpecificPreparation(argv) {
...
             "    $Ole32 = Add-Type -MemberDefinition '[DllImport(\"Ole32.dll\")]public static extern int CoAllowSetForegroundWindow(IntPtr pUnk, IntPtr lpvReserved);' -Name 'Ole32' -Namespace 'Win32' -PassThru\n" +
             "    $Ole32::CoAllowSetForegroundWindow([System.Runtime.InteropServices.Marshal]::GetIUnknownForObject($appActivator), [System.IntPtr]::Zero)",
             appUtilsPath
         );
     }
 
+    // write current time to a file to use it when gathering logs
+    var now = new Date();
+    var iso = now.toISOString();
+    fs.writeFileSync('startTime.txt', iso, util.DEFAULT_ENCODING);
```
Comment (robpaveza) [inline medic/medic-run.js]: Are we definitely writing to a file "startTime.txt"?  That seems like it'd be repeatedly overwritten and we'd lose data.

File: medic/medic-run.js [hunk 2]
```diff
@@ -232,12 +233,18 @@ function windowsSpecificPreparation(argv) {
             /^\s*\$appActivator .*$/gim,
             "$&\n" +
```
Comment (robpaveza) [inline medic/medic-run.js]: Mixing PS and JS in this way === gross.

File: medic/medic-run.js [hunk 3]
```diff
@@ -222,6 +222,7 @@ function windowsSpecificPreparation(argv) {
         var appUtilsPath   = path.join(libPath, "WindowsStoreAppUtils.ps1");
         var srcScriptPath  = path.join(CORDOVA_MEDIC_DIR, "lib", "patches", "EnableDebuggingForPackage.ps1");
         var destScriptPath = path.join(libPath, "EnableDebuggingForPackage.ps1");
+        var logScriptPath  = path.join(CORDOVA_MEDIC_DIR, "lib", "patches", "EnableLogging.ps1");
```
Comment (dblotsky) [inline medic/medic-run.js]: Should this be part of the cordova-windows platform template?

File: medic/medic-run.js (hunk 2 above)
Comment (dblotsky) [inline medic/medic-run.js]: This was copied over from the old Medic code. Having said that, improvements are in order at some point.

File: medic/medic-run.js (hunk 1 above)
Comment (dblotsky) [inline medic/medic-run.js]: This file is not necessary. It serves only to pass arguments from `medic-run` to `medic-log`. Instead, please pass a start time to medic-log on the command line from `cordova.conf`. You can do this by setting a `Property` using `SetPropertyFromCommand`, and then passing it to the logging step.

**Pull request closed without merging**